lib/WebpackConfig.js

```diff
@@ -97,8 +97,18 @@ class WebpackConfig {
         this.useSourceMaps = false;
         this.cleanupOutput = false;
         this.extractCss = true;
-        this.useImagesLoader = true;
-        this.useFontsLoader = true;
+        this.imageRuleOptions = {
+            type: 'asset',
+            maxSize: null,
```

Comment:
I'm wondering if we shouldn't override the default value for maxSize.

Based on the documentation everything under 8kb will be inlined... which isn't necessarily a bad thing but it will probably confuse some people/break things (for instance if someone used to import images in order to trigger a copy).

Reply:
> which isn't necessarily a bad thing but it will probably confuse some people/break things

I think this will also cause some confusion. On the other hand, if you have:

```js
import image from './images.foo.png';

img.src = image;
```

I think this will work regardless of whether it was inlined or copied. Unless we defaulted this to 0 (or used `asset/resource` instead of `assets` by default), this will surprise people. 

So, I'm conflicted: on the one hand, the inlining of 8kb and lower is probably better for performance... but will cause more confusion.

Reply:
I'm going to go the "more conservative" route and use `asset/resource` by default